randovania/games/fusion/exporter/patch_data_factory.py

```diff
@@ -216,6 +218,65 @@ def _create_nav_text(self) -> dict:
             }
         return nav_text_json
 
+    def _credits_elements(self) -> dict:
+        elements = {}
```

Comment:
Why not use `get_locations_for_major_pickups_and_keys`?

Reply:
Assume I just don't know it exists. How would I call it self.something.get_loc...?

Reply:
It's a function located in randovania/exporter/hints/credits_spoiler.py

Generally when function names are dropped and you don't quite know where they are, you can use your IDE to search the codebase for them.

Reply:
I'm not sure what to do differently with this, I got the data from the function but it's so much data that I don't seem to need. Is this a blocker to figure this out and incorporate?